tests/qinfo/test_fisher.py

```diff
@@ -412,9 +412,9 @@ def circ(x, y, z):
                 qml.RZ(zi, wires=1)
             return qml.probs(wires=range(n_wires))
 
-        x = [tf.Variable(np.pi / 8) for _ in range(2)]
-        y = [tf.Variable(np.pi / 8) for _ in range(10)]
-        z = [tf.Variable(1.0) for _ in range(1)]
+        x = tf.Variable(np.pi / 8 * np.ones(2), trainable=True)
```

Comment:
Ah nice!

Reply:
Yep, for the other interface tests (and now TF as well), we use a big tensor instead of a list of small tensors.